libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -206,6 +206,8 @@ singleton AP_Vehicle method get_control_output boolean AP_Vehicle::ControlOutput
 singleton AP_Vehicle method get_target_location boolean Location'Null
 singleton AP_Vehicle method set_target_velocity_NED boolean Vector3f
 singleton AP_Vehicle method set_target_angle_and_climbrate boolean float -180 180 float -90 90 float -360 360 float -FLT_MAX FLT_MAX boolean float -FLT_MAX FLT_MAX
+singleton AP_Vehicle method get_circle_radius boolean float'Null
+singleton AP_Vehicle method set_circle_rate boolean float -90 90
```

Comment:
I have used up 150 deg/s https://youtu.be/m5bwoiHsn8M?t=10

+- 360 should be enough, but I would be quite tempted not to check at all, the vehicle will still be limited by `WP_NAV_SPEED`

Reply:
ok, I've removed the limit on the rate that can be set.